src/vend-number.js

```diff
@@ -113,6 +113,34 @@ VendNumber.divide = function (...values) {
 }
 
 /**
+ * Returns the sum of all items in the collection based on property.
+ *
+ * @method sumBy
+ *
+ * @param {Collection} collection
+ *        A collection of items to loop through.
+ *
+ * @param {String} property
+ *        A property name to use for calculating the sum.
+ *
+ * @param {Number} decimalPoints
```

Comment:
This can be flagged as optional (with square brackets `[decimalPoints]`) as leaving it undefined will default to 2.d.p in `VendNumber.round()`